ddtrace/llmobs/_integrations/google_genai_utils.py

```diff
@@ -83,7 +85,7 @@ def extract_content(content):
     return [extract_content(contents)]
 
 
-def extract_metrics_google_genai(response) -> Dict[str, Any]:
+def extract_generate_metrics_google_genai(response) -> Dict[str, Any]:
```

Comment:
Maybe worth renaming this? `extract_generate_metrics` sounds not super clear. Maybe `extract_generation_metrics_...`?